Upstream-Status: Backport [https://github.com/rhinstaller/anaconda/commit/4d2164c]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

From 4d2164c19bb082dcff57df39556169ed484fa668 Mon Sep 17 00:00:00 2001
From: Vojtech Trefny <vtrefny@redhat.com>
Date: Thu, 1 Feb 2024 14:37:55 +0100
Subject: [PATCH] Do not use stringize and unicodeize from Blivet

These functions exist only for Python2/3 compatibility and will be
removed in the next Blivet release and Anaconda doesn't support
Python 2 anyway.
---
 pyanaconda/modules/storage/partitioning/specification.py | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/pyanaconda/modules/storage/partitioning/specification.py b/pyanaconda/modules/storage/partitioning/specification.py
index 360b1f2bc7..37f6a164f2 100644
--- a/pyanaconda/modules/storage/partitioning/specification.py
+++ b/pyanaconda/modules/storage/partitioning/specification.py
@@ -18,7 +18,6 @@
 # Red Hat Author(s): Chris Lumens <clumens@redhat.com>
 #
 
-from blivet.util import stringize, unicodeize
 from pykickstart.constants import AUTOPART_TYPE_PLAIN, AUTOPART_TYPE_BTRFS, AUTOPART_TYPE_LVM, \
     AUTOPART_TYPE_LVM_THINP
 
@@ -136,10 +135,7 @@ class PartSpec(object):
         return scheme == AUTOPART_TYPE_BTRFS and self.btr
 
     def __str__(self):
-        return stringize(self._to_string())
-
-    def __unicode__(self):
-        return unicodeize(self._to_string())
+        return self._to_string()
 
     def __eq__(self, other):
         return isinstance(other, PartSpec) and vars(self) == vars(other)
-- 
2.47.1

